File: F1BroadcastTools.Client/src/app/shared/models/Minimap.ts
export interface Minimap {
    trackId: number;
    cars: MinimapCar[];
    spectatorCarIdx: number;
    rotation?: number;
}

export interface MinimapCar {
    vehicleIdx: number;
    x: number;
    y: number;
    rotation: number;
    teamColor: string;
}

File: F1GameDataParser/ViewModels/Minimap/Minimap.cs
using F1GameDataParser.Enums;

namespace F1GameDataParser.ViewModels.Minimap
{
    public class Minimap
    {
        public Track TrackId { get; set; }
        public IEnumerable<MinimapCar> Cars { get; set; }
        public byte SpectatorCarIdx { get; set; }
        public int? Rotation { get; set; }
    }
}
